src/FilterConditionSelector/FilterConditionSelector.tsx

```diff
@@ -46,7 +46,14 @@ const FilterConditionSelector: React.FC<
 	const listRef = React.createRef<HTMLUListElement>();
 	const mainRef = React.useRef<HTMLElement | null>(null);
 	const [searchTerm, setSearchTerm] = React.useState(props.searchTerm);
+	const [
+		firstItemTree,
+		setFirstItemTree
+	] = React.useState<HTMLDivElement | null>(null);
 	const debouncedSearchTerm = useDebounce(searchTerm, 500);
+	let markedFirstItemTree: () => void;
```

Comment:
Если это функция, то нужен глагол
```suggestion
	let markFirstItemTree: () => void;
```

Reply:
И нужен ли `let`? Мы где-то переписываем значение?

Reply:
Да, значение переписываем на 233 строке